TESTING.md

```diff
@@ -0,0 +1,37 @@
+### Testing
```

Comment:
Can we make this more visible? Seems buried atm. Eg. link from the main readme or add to the beginning of contributing.

Reply:
Updated; I integrated this document in the `CONTRIBUTING.md` - let me know what you think 👍